doc/source/FAQ.rst

```diff
@@ -20,6 +20,23 @@ top-10 prediction for each user.
     :name: top_n_recommendations.py
     :lines: 10-
 
+.. _precision_recall_at_k:
+
+How to evaluate an algorithm using precision and recall at k metrics
+-----------------------------------------------------------------------
+
+Below is an example on how to compute Precision@k and Recall@k metrics for
+a recommendation model. We first split our dataset into train and test 
+folds. Using cross validation and on each iteration we train on our trainset 
+and get the predictions from the testset. Finally we compute Precision@k 
+and Recall@k for each user in predictions.
```

Comment:
I have changed the description as you suggested. The latex syntax you gave me is correct!
Note however that I have changed 
> An item is considered relevant if its rating is greater than the threshold k.

to 

> An item is considered relevant if its rating is greater than the user defined threshold. 

That's because the parameter `k` is irrelevant for this definition. You might have put it by mistake. I changed the definition of recommended similarly.